### DISCUSSION TITLE
Fujitsu 29f033 programming

### DISCUSSION THREAD

**Discussion by VamPikmin** [sanni/cartreader#1053] [Need help]
![hp~2](https://github.com/user-attachments/assets/14bae0cb-e203-4510-98a4-9be3247ce556)
Is it possible to reprogram these once they're soldered in a cart
I am using dip36 to tsop40 adapter (type 2)
From the manual pin 38 is we
On the adapter it's bridged to something, possibly vcc? 
Currently, the flash is not detected
Should i remove the bridge and solder the pad to we on the snes cart like the macronix flash?
Thank you

**Comment by sanni** [comment]
Yes removing the bridge and connecting the left pad to SNES WR should work, make sure you are using a current version of the OSCR firmware as there was a bug with flashing hirom carts that only got fixed like 6 months ago.

See also here: https://github.com/sanni/cartreader/wiki/Reflashing-SNES-SFC-Repros#translations

**Comment by VamPikmin** [comment]
Thank you, tested and working
